src/expr/src/explain/text.rs

```diff
@@ -728,18 +728,27 @@ impl MirRelationExpr {
             } => {
                 FmtNode {
                     fmt_root: |f, ctx: &mut PlanRenderingContext<'_, MirRelationExpr>| {
-                        if aggregates.len() > 0 {
-                            write!(f, "{}Reduce", ctx.indent)?;
-                        } else {
+                        if aggregates.len() == 0 && !ctx.config.raw_syntax {
                             write!(f, "{}Distinct", ctx.indent)?;
-                        }
-                        if group_key.len() > 0 {
+
                             if let Some(cols) = input.column_names(ctx) {
                                 let group_key = HumanizedExpr::seq(group_key, Some(cols));
-                                write!(f, " group_by=[{}]", separated(", ", group_key))?;
+                                write!(f, " on=[{}]", separated(", ", group_key))?;
```

Comment:
`Distinct on=...` might also be a bit misleading: the reader my think of the `DISTINCT ON` SQL syntax, which has a slightly different meaning: it doesn't project the specified columns, but performs a Top1.

How about `Distinct project=...` ? This would communicate that we project the specified columns and then do a distinct on the remaining thing.

Reply:
Done.